src/main/java/com/esri/samples/na/find_service_areas_for_multiple_facilities/README.md

```diff
@@ -1,38 +1,42 @@
-# Find service areas for multiple facilities
+<h1>Find service areas for multiple facilities</h1>
 
-Find the service areas of several facilities from a feature service.
+<p>Find the service areas of several facilities from a feature service.</p>
 
-![](FindServiceAreasForMultipleFacilities.png)
+<img src="FindServiceAreasForMultipleFacilities.png"/>
 
-## Use case
+<h2>Use case</h2>
 
-A service area is a region which can be accessed from a facility as limited by one or more factors, such as travel time, distance, or cost. When analyzing the service area of multiple facilities, this workflow can be used to identify gaps in service area coverage, or significant overlaps, helping to optimize the distribution of facilities. For example, a city's health service may identify areas of a city that can be served effectively from particular hospitals, and with this optimize distribution of staff and resources.
+<p>A service area is a region which can be accessed from a facility as limited by one or more factors, such as travel time, distance, or cost. When analyzing the service area of multiple facilities, this workflow can be used to identify gaps in service area coverage, or significant overlaps, helping to optimize the distribution of facilities. For example, a city's health service may identify areas of a city that can be served effectively from particular hospitals, and with this optimize distribution of staff and resources.</p>
 
-## How to use the sample
+<h2>How to use the sample</h2>
 
-Click 'find service areas' to calculate and display the service area of each facility on the map. The polygons displayed around each facility represents the service area that is within 3 minutes away from the hospital by car.
+<p>Click 'find service areas' to calculate and display the service area of each facility on the map. The polygons displayed around each facility represents the service area; in red is the area that is within 1 minute away from the hospital by car. Light orange is the area that is within 3 minutes away from the hospital by car. All polygons have transparency to them, so that when service areas from multiple facilities overlap, these areas will appear in bolder colors.</p>
```

Comment:
can shorten this a bit:

"The polygons displayed around each facility represents the _facility's_ service area; the red area  is within 1 minute travel time from the hospital by car, whilst orange is within 3 minutes by car. All polygons are transparent, so that when service areas from multiple facilities overlap, these areas will appear as bolder colors."